pandasai/__init__.py

```diff
@@ -627,7 +629,10 @@ def run_code(
 
         # Add save chart code
         if self._save_charts:
-            code = add_save_chart(code, self._prompt_id, not self._verbose)
+
+            code = add_save_chart(code, self._prompt_id,
```

Comment:
Can you please double check you're running the code through `black`? I think this function call could be formatted to be a little more readable.

Reply:
Will do it